amr-wind/physics/ConvectingTaylorVortex.H

```diff
@@ -33,6 +33,54 @@ struct VExact
     const int m_comp{1};
 };
 
+struct WExact
+{
+    AMREX_GPU_HOST_DEVICE AMREX_FORCE_INLINE amrex::Real operator()(
+        const amrex::Real,
+        const amrex::Real,
+        const amrex::Real,
+        const amrex::Real,
+        const amrex::Real,
+        const amrex::Real) const;
+    const int m_comp{2};
+};
+
+struct gpxExact
```

Comment:
Can we switch these names to be consistent with how we define struct/class in AMR-Wind? 

```suggestion
struct GpxExact
```

Reply:
thanks, fixed it